setup.cfg

```
@@ -93,3 +93,10 @@ console_scripts =
     ssst = ssst.cli:cli
 pytest11 =
     ssst = ssst._tests.pytest_plugin
+
+[versioneer]
+VCS = git
+style = pep440
+versionfile_source = src/ssst/_version.py
+versionfile_build = ssst/_version.py
+tag_prefix = ''
```

```suggestion
tag_prefix = 'v'
```
(similar to https://github.com/epcpower/sunspec-demo/blob/81be80cb68bf0019a1478c3693e0c5b7883ab648/setup.cfg#L47)

---

\*bump\* here.  Also, hopefully the build will pass with an update merge from `main` (optionally using the button down this page near the merge button).

---

Did we reach a conclusion on the choice of using a tag prefix of `v` vs `release/` ? That's why I was holding on this change request.

---

Sorry, I totally forgot that whole conversation.  I restarted in slack and will summarize here when we finish it.

---

RTD offers no apparent control and seems to 'expect' a bare version number based on their `v: <exact tag>` combo box for version selection.  Since I have no predicted other use for tags...  whatever.  Leave it as `''` because or switch to `'v'` for consistency.  No big deal.

---

My conclusion is to leave as is, with no tag_prefix. Versions will be in the format `1.2.3`.